src/style.css

```
@@ -27,6 +27,12 @@
   margin-top: .75em
 }
 
+/* Some flags (e.g. Guam's) contain identifying words that can give away the answer */
+/* Card backs always show the unmodified image(s) */
+/* If a blurred image(s) is available, card front shows that instead */
+.image--front > img[src*="-blur"] + img { display: none; }
+.image--back > img[src*="-blur"] { display: none; }
```

I'm not sure about this, since this is the way @axelboc had originally suggested it, and it is nice and concise, but I think that it might be more readable to split each of these into three lines, especially since that's the convention we follow elsewhere in `style.css`. In any case, it's just a question of formatting.
```suggestion
.image--front > img[src*="-blur"] + img {
  display: none;
}
.image--back > img[src*="-blur"] {
  display: none;
}
```

---

My vote would be to style everything automatically. [This StackOverflow answer](https://stackoverflow.com/a/55443013/6047827) suggests using [Prettier](https://prettier.io/) with [Husky](https://github.com/typicode/husky) to "set up a pre-commit hook which will automatically clean your files using Prettier before committing any new code."

It might not be worth it to set up the hook for it to happen automatically, but a trivially easy option for now would be to manually paste code into https://prettier.io/playground, click *Show options* in the bottom left to change the parser to *css*, and paste in the code to be formatted. That changes the styling to match what you recommend.

If the project were to adopt this as a general policy, note it would make this change at the bottom of the `style.css` file. There are configuration options, but I'm not sure whether they are all available from the online playground. 

*Changing from*
```css
.nightMode .info, .nightMode .type {
  color: #ccc;
}
```

*Changing to*
```css
.nightMode .info,
.nightMode .type {
  color: #ccc;
}
```

---

Yep, let's do that. I'm a big fan of Prettier! I'll open a new issue to set up Prettier. In the meantime, just use the playground as you suggested. Thanks!

---

> If the project were to adopt this as a general policy, note it would make this change at the bottom of the style.css file.

That makes sense!